### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by Lucas-ConU**
Login, logout and sign up implementation
Different pages set up
Database stores user's information
Base html coding for the default pages 
Issues closed: #6 , #11 , #12 and #5

### CODE REVIEW

File: backend/backend/urls.py [hunk 1]
```diff
@@ -13,9 +13,14 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.contrib import admin
-from django.urls import path
-
+from django.conf.urls import url, include
+from pigeonpost import views as core_views
+from django.contrib.auth.views import LoginView, LogoutView
```
Comment (Ahmad-ConU) [inline backend/backend/urls.py:18]: auth.views not needed

**Review by seaiam — APPROVED** [review]
Looks good ! I'm able to sign up , log in and log out .

**Review by hakimsheriff — APPROVED** [review]
Looks good! I am able to create a new user and login as that user. This meets all criteria for relevant issues.

File: backend/backend/urls.py (hunk 1 above)
Comment (Ahmad-ConU) [reply backend/backend/urls.py:18]: Functionality of sign up works just fine

**Pull request merged**